<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditAdminDialog</class>
 <widget class="QWidget" name="EditAdminDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1455</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#EditAdminDialog{
    background-color : #FFFFFF;
}

*{
    font-family : 微软雅黑;
    font-size : 14px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="bg" native="true">
     <property name="styleSheet">
      <string notr="true">#bg{
    background-color : rgba(0, 0, 0, 0.5);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>496</number>
      </property>
      <property name="topMargin">
       <number>164</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="content" native="true">
        <property name="minimumSize">
         <size>
          <width>458</width>
          <height>532</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>458</width>
          <height>532</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#content{
    background-color : #FFFFFF;
    border-radius : 20px;
}</string>
        </property>
        <widget class="QLabel" name="tittleLabel">
         <property name="geometry">
          <rect>
           <x>181</x>
           <y>20</y>
           <width>96</width>
           <height>21</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#tittleLabel{
    font-size : 16px;
    font-weight : 900;
    line-height : 21px;
}</string>
         </property>
         <property name="text">
          <string>新增后台用户</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="starLabel">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>86</y>
           <width>16</width>
           <height>19</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#starLabel{
    line-height : 19px;
    color : #FF4A3C;
    font-weight : 900;
}</string>
         </property>
         <property name="text">
          <string>*</string>
         </property>
        </widget>
        <widget class="QLabel" name="emailLabel">
         <property name="geometry">
          <rect>
           <x>38</x>
           <y>86</y>
           <width>77</width>
           <height>19</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#emailLabel{
    line-height : 19px;
    font-weight : 900;
}</string>
         </property>
         <property name="text">
          <string>用户邮箱</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="emailEdit">
         <property name="geometry">
          <rect>
           <x>122</x>
           <y>72</y>
           <width>306</width>
           <height>48</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#emailEdit{
     background-color : #F7F7F7;
     border : none;
     border-radius : 8px;
     color : #999999;
     line-height : 14px;
     padding-left : 16px;
     placeholder-text-color : #999999;
}</string>
         </property>
         <property name="placeholderText">
          <string>请输入用户邮箱</string>
         </property>
        </widget>
        <widget class="QLabel" name="roleLabel">
         <property name="geometry">
          <rect>
           <x>38</x>
           <y>154</y>
           <width>77</width>
           <height>19</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#roleLabel{
    line-height : 19px;
    font-weight : 900;
}</string>
         </property>
         <property name="text">
          <string>用户角色</string>
         </property>
        </widget>
        <widget class="QLabel" name="starLabel2">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>154</y>
           <width>16</width>
           <height>19</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#starLabel2{
    line-height : 19px;
    color : #FF4A3C;
    font-weight : 900;
}</string>
         </property>
         <property name="text">
          <string>*</string>
         </property>
        </widget>
        <widget class="QComboBox" name="roleComboBox">
         <property name="geometry">
          <rect>
           <x>122</x>
           <y>140</y>
           <width>306</width>
           <height>48</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox{
     background-color : #F7F7F7;
     border-radius : 8px;
     color : #999999;
     padding-left : 16px;
     placeholder-text-color : #999999;
}

QComboBox::drop-down{
     border:none;
     width : 33px;
}

QComboBox::down-arrow{
    width : 24px;
    height : 24;
    background-color : transparent;
    image : url(&quot;:/images/admin/triangle2.png&quot;);
}

QComboBox QAbstractItemView{
    outline : 0px;
}


QComboBox QAbstractItemView::item{
    color : #222222;
    padding-left : 16px;
    background-color : #FFFFFF;
    height : 30px;
}

QComboBox QAbstractItemView::item:selected
{
     background-color : #409CE1;
     color : #FFFFFF;
}

</string>
         </property>
         <property name="placeholderText">
          <string>请选择用户角色</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="nameEdit">
         <property name="geometry">
          <rect>
           <x>122</x>
           <y>208</y>
           <width>306</width>
           <height>48</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#nameEdit{
    background-color : #F7F7F7;
    border : none;
    border-radius : 8px;
    color : #999999;
    line-height : 19px;
    padding-left : 16px;
    placeholder-text-color : #999999;
}</string>
         </property>
         <property name="placeholderText">
          <string>请输入用户昵称</string>
         </property>
        </widget>
        <widget class="QLabel" name="starLabel3">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>222</y>
           <width>16</width>
           <height>19</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#starLabel3{
    line-height : 19px;
    color : #FF4A3C;
    font-weight : 900;
}</string>
         </property>
         <property name="text">
          <string>*</string>
         </property>
        </widget>
        <widget class="QLabel" name="nameLabel">
         <property name="geometry">
          <rect>
           <x>38</x>
           <y>222</y>
           <width>77</width>
           <height>19</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#nameLabel{
    line-height : 19px;
    font-weight : 900;
}</string>
         </property>
         <property name="text">
          <string>用户昵称</string>
         </property>
        </widget>
        <widget class="QLabel" name="statusLabel">
         <property name="geometry">
          <rect>
           <x>38</x>
           <y>290</y>
           <width>77</width>
           <height>19</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#statusLabel{
    line-height : 19px;
    font-weight : 900;
}</string>
         </property>
         <property name="text">
          <string>用户状态</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="startBtn">
         <property name="geometry">
          <rect>
           <x>122</x>
           <y>288</y>
           <width>61</width>
           <height>23</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QRadioButton{
    color : #222222;
}

QRadioButton::checked{
    color : #3ECEFF;
}

QRadioButton::indicator{
    width : 16px;
    height : 16px;
}

QRadioButton::indicator::checked{
    image : url(&quot;:/images/admin/checked.png&quot;);
}

QRadioButton::indicator::unchecked{
    image : url(&quot;:/images/admin/unchecked.png&quot;);
}
</string>
         </property>
         <property name="text">
          <string>启用</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QRadioButton" name="stopBtn">
         <property name="geometry">
          <rect>
           <x>205</x>
           <y>288</y>
           <width>61</width>
           <height>23</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QRadioButton{
    color : #222222;
}

QRadioButton::checked{
    color : #3ECEFF;
}

QRadioButton::indicator{
    width : 16px;
    height : 16px;
}

QRadioButton::indicator::checked{
    image : url(&quot;:/images/admin/checked.png&quot;);
}

QRadioButton::indicator::unchecked{
    image : url(&quot;:/images/admin/unchecked.png&quot;);
}
</string>
         </property>
         <property name="text">
          <string>停用</string>
         </property>
        </widget>
        <widget class="QLabel" name="remarkLabel">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>330</y>
           <width>69</width>
           <height>19</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#remarkLabel{
     font-weight : 900;
     line-height : 19px;
     color : #222222;
}</string>
         </property>
         <property name="text">
          <string>备注</string>
         </property>
        </widget>
        <widget class="QPlainTextEdit" name="commentTextEdit">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>360</y>
           <width>398</width>
           <height>84</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#commentTextEdit{
    background-color : #F7F7F7;
    border : none;
    outline : none;
    border-radius : 8px;
    color : #222222;
    padding : 14px 16px 14px 16px;
}</string>
         </property>
         <property name="placeholderText">
          <string>请输入内容</string>
         </property>
        </widget>
        <widget class="QLabel" name="wordCount">
         <property name="geometry">
          <rect>
           <x>385</x>
           <y>415</y>
           <width>41</width>
           <height>19</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#wordCount{
    color : #999999;
}</string>
         </property>
         <property name="text">
          <string>0/10</string>
         </property>
        </widget>
        <widget class="QPushButton" name="cancelBtn">
         <property name="geometry">
          <rect>
           <x>114</x>
           <y>474</y>
           <width>100</width>
           <height>38</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#cancelBtn{
    background-color : #FFFFFF;
    border : 1px solid #DEDEDE;
    border-radius : 8px;
    color : #999999;
    font-size : 16px;
}</string>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
        <widget class="QPushButton" name="submitBtn">
         <property name="geometry">
          <rect>
           <x>244</x>
           <y>474</y>
           <width>100</width>
           <height>38</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">#submitBtn{
    background-color : #3ECEFF;
    border : 1px solid #3ECEFF;
    border-radius : 8px;
    color : #FFFFFF;
    font-size : 16px;
}</string>
         </property>
         <property name="text">
          <string>提交</string>
         </property>
        </widget>
        <zorder>tittleLabel</zorder>
        <zorder>starLabel</zorder>
        <zorder>emailLabel</zorder>
        <zorder>emailEdit</zorder>
        <zorder>starLabel2</zorder>
        <zorder>roleComboBox</zorder>
        <zorder>roleLabel</zorder>
        <zorder>nameEdit</zorder>
        <zorder>starLabel3</zorder>
        <zorder>nameLabel</zorder>
        <zorder>statusLabel</zorder>
        <zorder>startBtn</zorder>
        <zorder>stopBtn</zorder>
        <zorder>remarkLabel</zorder>
        <zorder>commentTextEdit</zorder>
        <zorder>wordCount</zorder>
        <zorder>cancelBtn</zorder>
        <zorder>submitBtn</zorder>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
